package.json

```diff
@@ -30,10 +30,10 @@
     "remark": "13.0.0",
     "remark-html": "13.0.1",
     "styled-components": "^5.3.3",
-    "tinacms": "^0.68.12"
+    "tinacms": "latest"
```

Comment:
@jamespohalloran This should probably be a real version and not a tag. This is so that the generated files can change when we have version bumps (as well if we have any breaking changes. )